src/pages/default-detail-page.js

```diff
@@ -25,7 +25,12 @@
 		var onFragmentChanged = function (key, data) {
 			var frag = !!data ? data : '';
 
-			changer.navigateTo(frag.split('/')[0]);
+			if (!!isFirstFragment) {
+				changer.navigateTo(frag.split('/')[0]);
+			} else {
+				isFirstFragment = false;
+			}
+
```

Comment:
So the variable was declared, but never used ?? How come jshint did not find it ?

Reply:
jshint on vscode detected it.